src/modules/core/components/Slider/Slider.css

```
@@ -17,3 +17,11 @@
 .danger :global(.rc-slider-handle:active) {
   box-shadow: 0 0 5px var(--danger);
 }
+
+/*
+  Removes `left` property from the dot for the correct visual position.
+  I tried many options but that's the only one I found working
+*/
+.primary > div > div > span {
+  left: 0 !important;
+}
```

I think this also works:
```suggestion
.primary :global(.rc-slider-dot) {
  left: 0 !important;
}
```

The advantages of my version, is that you can be sure it will still work even if some updates will change the html element structure